README.md

```diff
@@ -1,130 +1,61 @@
 # Mattermost Bitbucket Plugin
 
+[![Build Status](https://github.com/mattermost/mattermost-plugin-bitbucket/actions/workflows/ci.yml/badge.svg)](https://github.com/mattermost/mattermost-plugin-bitbucket/actions/workflows/ci.yml)
+[![Code Coverage](https://img.shields.io/badge/Code%20Coverage-%-brightgreen?style=flat-square)](https://github.com/mattermost/mattermost-plugin-bitbucket)
```

Comment:
The icon is broken
![Screenshot from 2023-10-30 09-06-56](https://github.com/mattermost/mattermost-plugin-bitbucket/assets/16541325/7d3d78d6-8995-4ce5-9eeb-12a47412f989)

Reply:
@hanzei I'll verify that but still, this is a placeholder because I do not have access to any code coverage data

Reply:
I'd suggest to remove it since there is no data to add about code coverage

Reply:
To explain better, I just wanted to add a placeholder for this badge. I do not have the data to show here. What is more, I have no idea why you see the broken icon because it's working for me 
![image](https://github.com/mattermost/mattermost-plugin-bitbucket/assets/17500766/e8f73f35-9e6f-4b12-8ef3-cb332242d3d9)
So maybe removing is the best option now

Reply:
Yes, please remove it. Thanks!

Reply:
@cwarnermm just pushed new version after deletion